src/sql/workbench/browser/parts/views/customView.ts

```diff
@@ -700,7 +700,7 @@ class TreeDataSource implements IAsyncDataSource<ITreeItem, ITreeItem> {
 
 	getChildren(node: ITreeItem): Promise<any[]> {
 		if (node.childProvider) {
-			return this.withProgress(this.objectExplorerService.getChildren(node, this.id)).catch(e => {
+			return this.withProgress(this.objectExplorerService.refreshNode(node, this.id)).catch(e => {
```

Comment:
I would rather hide these details in the oeshim service. Rather than a caller having to explicitly know to call refreshNode vs getchildren, the oeshim services should just do the refresh node as part of the getchildren call if it needs to refresh the node (it already knows about the node).